lib/shoryuken/fetcher.rb

```diff
@@ -32,7 +32,7 @@ def fetch(queue, available_processors)
           limit = batch ? FETCH_LIMIT : available_processors
 
           if (sqs_msgs = Array(receive_messages(queue, limit))).any?
-            logger.info { "Found #{sqs_msgs.size} messages for '#{queue}'" }
+            logger.debug { "Found #{sqs_msgs.size} messages for '#{queue}'" }
```

Comment:
@mariokostelac WDYT about it? I kind of like this change, but it can mislead people that were used to these messages.